tests/fault_tolerance/migration/test_sglang.py

```
@@ -94,9 +112,17 @@ def __init__(self, request, worker_id: str, migration_limit: int = 3):
         )
 
-    def get_pid(self):
-        """Get the PID of the worker process"""
-        return self.proc.pid if self.proc else None
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        """Release allocated port when worker exits."""
+        try:
+            # Release allocated port BEFORE calling parent __exit__
+            if hasattr(self, "system_port") and self.system_port is not None:
+                free_port(self.system_port)
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)